contracts/EtherTimelock.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.4.17;
+
+/**
+ * @title EtherTimelock
+ * @dev EtherTimelock is a contract that will allow a
+ * beneficiary to extract ether sent to it after a given release time
+ */
+contract EtherTimelock {
+    //beneficiary of funds
+    address public beneficiary;
+
+    //timestamp when funds are able to be released
+    uint64 public releaseTime;
+
+    function EtherTimelock(address _beneficiary, uint64 _releaseTime) {
+        require(_releaseTime > now && _beneficiary != address(0));
```

Comment:
thx for adding it, but we are not using `&&` conditions inside the requires, can you separate them in two lines please?

Reply:
no problem! is that just for readability or is there another reason for that?

Reply:
Readability; it's also more explicit, and the compiler doesn't really are about the difference.